CHANGES.md

```diff
@@ -1,3 +1,7 @@
+## v0.1.3
+
+* Fix some dashboard UI elements not rendering correctly.
```

Comment:
This is a minor bugfix for some web dashboard UI elements that were not rendering correctly.